lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -173,8 +174,13 @@ function createBucket(authInfo, bucketName, headers,
         null, null, null, null, null, null, null, null,
         null, isNFSEnabled);
 
-    if (locationConstraint !== undefined) {
-        bucket.setLocationConstraint(locationConstraint);
+    const { locationConstraintStr, isIngestion } =
+        locationConstraint.split(zenkoSeparator);
+    if (locationConstraintStr !== undefined) {
+        bucket.setLocationConstraint(locationConstraintStr);
+    }
+    if (isIngestion !== undefined) {
+        bucket.setIsIngestion(isIngestion);
```

Comment:
and here, only if we matched the `ingest` suffix, set it to a hardcoded `true`

Reply:
the 'ingest' check is in the `BucketInfo` model in Arsenal, which expects `null`, `ingest`, or another different value to pause ingestion. Would appreciate any suggestions on what kind of value should be expected to set the ingestion status to disabled!

Reply:
I just talked to @rahulreddy, this may stem from my vague naming - I will change it to just `ingestion`, since `isIngestion` implies a true/false value.

We're currently storing the ingestion value as `null`, `{ status: 'Enabled' }`, and `{ status: 'Disabled' }`. They aren't going to be used just yet since we aren't implementing suspend yet, but we're trying to make it so when we include it in the future it'll be here.

Reply:
I think we intend it to be similar to `Paused`, so that backlogs can be processed. tagging @rahulreddy again to confirm

If that's the case, instead of `Disabled`, I should have `status` as `Paused` instead?